scripts/zones/Mamook/Zone.lua

```diff
@@ -8,6 +8,9 @@ local zone_object = {}
 
 zone_object.onInitialize = function(zone)
     xi.helm.initZone(zone, xi.helm.type.LOGGING)
+	-- Clones should not automatically spawn on server init
+	-- TODO: There's probably a better way to handle this in the db
+    for i = 1, 4 do DespawnMob(ID.mob.HUNDREDFACED_HAPOOL_JA + i) end
```

Comment:
You can change the spawn flags (I think in the groups or spawn points) to have them not spawn by default

Reply:
spawnType, set them to scripted spawns.